E/EGL/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "EGL"
+version = v"1.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.freedesktop.org/mesa/mesa.git", "1b74a12ea0ae900d49d1921ed8931eb6131e1f18")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+mkdir build
+cd build
+apk add py3-mako meson wayland-dev
+
+# pkgconfig returns the wrong path here. It's an ugly hack, but it works
```

Comment:
I'll have a look about what is going on there, but in any case you must delete this link at the end, otherwise it'll end up inside the tarball

Reply:
Ah, right! I forgot.